src/Uno.UI/UI/Xaml/FrameworkElement.Interface.Skia.cs

```diff
@@ -38,9 +38,25 @@ private protected virtual void OnUnloaded()
 
         }
 
-        public TransitionCollection Transitions { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
 
-        public object FindName(string name)
+		#region Transitions Dependency Property
+
+		[GeneratedDependencyProperty(DefaultValue = null, ChangedCallback = true)]
+		public static DependencyProperty TransitionsProperty { get; } = CreateTransitionsProperty();
+
+		public TransitionCollection Transitions
+		{
+			get => GetTransitionsValue();
+			set => SetTransitionsValue(value);
+		}
+
+		private void OnTransitionsChanged(DependencyPropertyChangedEventArgs args)
```

Comment:
Why generate a callback if private and empty?

Reply:
Will be improved in next PR.